clang/lib/CIR/CodeGen/CIRGenCXXABI.h

```diff
@@ -37,6 +37,12 @@ class CIRGenCXXABI {
 
   void setCXXABIThisValue(CIRGenFunction &cgf, mlir::Value thisPtr);
 
+  /// Emit the code to initialize hidden members required to handle virtual
+  /// inheritance, if needed by the ABI.
+  virtual void
+  initializeHiddenVirtualInheritanceMembers(CIRGenFunction &cgf,
+                                            const CXXRecordDecl *rd) {}
```

Comment:
Should this be pure virtual?  Or are we not expecting this to be implemented by 1 or more targets?

Reply:
The MSCXXABI overrides this method, but all other targets use this default implementation.